### PR TITLE
feat: Add integration test for NodeCreateTransaction

### PR DESCRIPTION
**Pull request by gsstoykov**
**Description**:
This PR serves as a POC for using NodeCreateTransaction.

**Related issue(s)**:

Fixes https://github.com/hashgraph/hedera-sdk-go/issues/1085

**Notes for reviewer**:
Can be merged even if POC as it adds an Integration test. CI will fail against `local-node` so the test will be disabled(It needs `Solo` configured in our CI).

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by gsstoykov** [comment]
Screenshot 2024-10-10 at 13 30 58

File: node_create_transaction_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// The test has to be disabled so it doesn't fail calls to local-node
+func DisableTestIntegrationCanExecuteNodeCreateTransaction(t *testing.T) {
+	t.Parallel()
+	env := NewIntegrationTestEnv(t)
```
Comment (0xivanov) [inline node_create_transaction_e2e_test.go:36]:
```suggestion
	env := NewIntegrationTestEnv(t)
	defer CloseIntegrationTestEnv(env, nil)
```

File: node_create_transaction_e2e_test.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		SetServiceEndpoints([]Endpoint{endpoint}).
+		SetGossipCaCertificate(validGossipCert).
+		SetAdminKey(adminKey).
+		Execute(env.Client)
+
+	_, err := resp.GetReceipt(env.Client)
+	require.NoError(t, err)
+
+	err = CloseIntegrationTestEnv(env, nil)
+	require.NoError(t, err)
```
Comment (0xivanov) [inline node_create_transaction_e2e_test.go:74]: remove

**Review by 0xivanov — APPROVED** [review]
LGTM

**Pull request merged**